src/olympia/addons/models.py

```diff
@@ -1793,6 +1793,23 @@ def git_extraction_is_in_progress(self):
 
         return GitExtractionEntry.objects.filter(addon=self, in_progress=True).exists()
 
+    @cached_property
+    def tag_list(self):
+        attach_tags([self])
+        return self.tag_list
```

Comment:
I... never thought of doing it that way (returning the same property in a `@cached_property` method) - before. Neat.